crypto/evp/p5_crpt2.c

```diff
@@ -21,15 +21,14 @@
 
 int ossl_pkcs5_pbkdf2_hmac_ex(const char *pass, int passlen,
                               const unsigned char *salt, int saltlen, int iter,
-                              const EVP_MD *digest, int keylen,
+                              const char *mdname, int keylen,
```

Comment:
I wonder if this change steps on @jon-oracle changes in p12?

Reply:
Possibly, the work around is fairly trivial: get the digest name.